test/e2e/provisioning/all_e2e_test.go

```diff
@@ -389,3 +390,22 @@ func TestDeploymentControllerUpgradesMachineE2E(t *testing.T) {
 	}
 	testScenario(t, scenario, *testRunIdentifier, params, HZManifest, false)
 }
+
+// TestCherryServersProvisioning - a test suite that exercises CherryServers provider
+// by requesting nodes with different combination of container runtime type, container runtime version and the OS flavour.
+func TestCherryServersProvisioningE2E(t *testing.T) {
```

Comment:
@nacef-labidi To make E2E tests work you need to update the tests suite to replace the project and team IDs with values from the environment.

Reply:
@xmudrii Are you defining env vars from your side or do I need to set them somewhere ?

Reply:
We are defining them from our side, you just need to replace `<< YOUR_PROJECT_ID >>` with the content of `CHERRYSERVERS_PROJECT_ID` environment variable and same for the team ID.

Reply:
Done